tests/models/test_taskmixin.py

```diff
@@ -287,7 +295,8 @@ def test_set_setup_teardown_ctx_dependencies_using_classic_tasks(dag_maker):
         t3 = make_task("t3", type_="classic")
         setuptask = make_task("setuptask", type_="classic", setup_=True)
         teardowntask = make_task("teardowntask", type_="classic", teardown_=True)
-        with setuptask >> teardowntask:
+        with setuptask >> teardowntask as scope:
+            scope.add_task(t1, t2, t3)
```

Comment:
Instead of doing this, can we not move the t1 t2 t3 creation inside the block?

Reply:
yeah but the whole point of these tests are the case where the task is instantiated out of context.  they were just testing the arrow-auto-add behavior.  so, now, we could probably just remove them, but they are there, and can't hurt, and `scope.add_task` is the way to achieve this now, so, seems appropriate to just update to use add_task and keepem.

Reply:
BUT... no strong opinion about it, just explaining why I used add_task

For now, i am signing off, feel free to make any changes

Reply:
Maybe we should just not support the use case (for 2.6 at least).

Reply:
You mean not support `add_task`?

Reply:
i.e. remove `add_task` from setupteardowncontext and taskgroup ?

Reply:
I am fine with that personally.

Reply:
you have my supprort for this, but now i go

Reply:
> yeah but the whole point of these tests are the case where the task is instantiated out of context. they were just testing the arrow-auto-add behavior. so, now, we could probably just remove them, but they are there, and can't hurt, and scope.add_task is the way to achieve this now, so, seems appropriate to just update to use add_task and keepem. i think the normal instantiated-in-context behavior is already tested elsewher.

I removed it. The tests are no longer needed because even the title would be wrong